tests/PHPStan/Rules/Methods/data/bug-727.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Bug727;
+
+class HelloWorld
+{
+	public static function sayHello(): void
+	{
+		echo 'Hello';
+	}
+
+	public function getClosure(): void
+	{
+		$closure1 = \Closure::fromCallable([self::class, 'sayHello']);
+		$closure2 = \Closure::fromCallable([static::class, 'sayHello']);
+		$closure2 = \Closure::fromCallable([static::class, 'sayHello2']);
+	}
+}
```

Comment:
This is a copy of bug-1871.php with fixes to prove that the correct method is discoverable

Reply:
Shoot, I should probably rename the file to be the issue number from https://github.com/phpstan/phpstan/issues/5782 versus this PR number. Or is it OK?

Reply:
Yeah, these test case numbers are always issues